### DISCUSSION TITLE
Chinese version of VueUse

### DISCUSSION THREAD

**Discussion by lzdFeiFei** [vueuse/vueuse#2081] [Show and tell]
I've create a repo for Chinese version of VueUse , but I just started it.

For now, I have translated the 'Guide' part,  I hope to finish the 'Core function' part in next two weeks.

https://vueuse-cn.github.io/vueuse-cn/

**Comment by sykes10** [comment]
Do you think we can add languages to the original repo? then people can collaborate with translations. I will like to add Spanish to this package.

**Reply by lzdFeiFei** [reply]
I'm not sure the author of repo will agree that. In some issue, they said there will not have offical translations. So I create a repo for Chinese version, and put it on github pages.

**Comment by linghengqian** [comment]
I'm just guessing that the reason the above is referring to is because currently vitepress doesn't officially handle i18n.
